### PR TITLE
unpin libvirt

### PR DESCRIPTION
**Pull request by SeanMooney**
- unpin the libvirt package

### CODE REVIEW

**Comment by slagle** [comment]
recheck

File: roles/edpm_libvirt/meta/argument_specs.yml [hunk 1]
```diff
@@ -20,14 +20,18 @@ argument_specs:
...
         description: The list of packages to install for libvirt.
         default:
-          - libvirt
-          - libvirt-admin
-          - libvirt-client
-          - libvirt-daemon
+          - libvirt {{edpm_libvirt_range}}
+          - libvirt-admin {{edpm_libvirt_range}}
+          - libvirt-client {{edpm_libvirt_range}}
+          - libvirt-daemon {{edpm_libvirt_range}}
```
Comment (slagle) [inline roles/edpm_libvirt/meta/argument_specs.yml:34]: the space between name and range looks odd here, but apparently this works with dnf. this worked in my local test:

`
sudo dnf install libvirt\ \<9.7.0-2
`

I guess the ansible module will handle this properly.

**Review by slagle — COMMENTED** [review]
+2

File: roles/edpm_libvirt/meta/argument_specs.yml (hunk 1 above)
Comment (SeanMooney) [reply roles/edpm_libvirt/meta/argument_specs.yml:34]: the ansible module example had a space when i looked at it which is why i added it here.

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html#examples


- name: Install Apache >= 2.4
  ansible.builtin.dnf:
    name: httpd >= 2.4
    state: present

the package module support the same syntax.
but it does not have an example so i kept the style form the dnf module


libvirt 10 is now available in the mirrors so we can proceed with unpinning the version

**Review by slagle — APPROVED** [review]

**Pull request merged**